packages/web-app-files/src/views/Personal.vue

```diff
@@ -203,16 +209,71 @@ export default {
   },
 
   methods: {
-    ...mapActions('Files', ['loadIndicators', 'loadPreview']),
+    ...mapActions('Files', ['loadIndicators', 'loadPreview'], 'showMessage'),
     ...mapMutations('Files', [
       'SELECT_RESOURCES',
       'SET_CURRENT_FOLDER',
       'LOAD_FILES',
       'CLEAR_CURRENT_FILES_LIST',
-      'UPDATE_CURRENT_PAGE'
+      'REMOVE_FILE',
+      'REMOVE_FILE_FROM_SEARCHED'
     ]),
     ...mapMutations(['SET_QUOTA']),
 
+    async fileDropped(fileIdTarget) {
+      const selected = this.selectedFiles
+      const targetInfo = this.activeFiles.find(e => e.id === fileIdTarget)
+      const isTargetSelected = selected.some(e => e.id === fileIdTarget)
+      if (isTargetSelected) return
+      if (targetInfo.type !== 'folder') return
+      const errors = []
+      const itemsInTarget = await this.fetchResources(targetInfo.path)
+
+      selected.forEach(async current => {
```

Comment:
Happens async, so we might get errors after we showed the current set of errors. I will take care of this.

Reply:
fixed in recent commit, switched to using a `p-queue` and awaiting `Promise.all`.